department-of-veterans-affairs/next-build
# feat: Enable storys feature flags for nextbuild prod.

# Description

Re-enable story and story listings for the eks flag envirionment


## Developer Task

```[tasklist]
```

## Testing Steps
I ran ddev locally and enabled both 
FEATURE_NEXT_BUILD_CONTENT_NEWS_STORY and FEATURE_NEXT_BUILD_CONTENT_STORY_LISTING 
then I was able to see the story page for http://localhost:3999/butler-health-care/stories/ which is needed by an alarm setup for preview-prod


# Reviewer

### Reviewing a PR

This section lists items that need to be checked or updated when making changes to this repository. 

## Standard Checks

```[tasklist]
```
